robotbase/src/main/java/frc/robot/commands/scoring/SpeakerScoreFromBase.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.commands.scoring;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants.*;
+import frc.robot.commands.intake.IntakeFeedToShooter;
+import frc.robot.commands.intake.IntakeStop;
+import frc.robot.commands.pivot.PivotSetRotation;
+import frc.robot.commands.shooter.ShooterSetRPMs;
+import frc.robot.commands.shooter.ShooterStop;
+import frc.robot.commands.shooter.ShooterWaitForRPMs;
+
+public class SpeakerScoreFromBase extends SequentialCommandGroup {
+  public SpeakerScoreFromBase() {
+    super(
+        new ParallelDeadlineGroup(
+            new PivotSetRotation(PIVOT.SPEAKER_SCORE_FROM_BASE_ROTATION),
+            new ShooterSetRPMs(
+                SHOOTER.TOP_MOTOR_SPEAKER_SCORE_FROM_BASE_RPMS,
+                SHOOTER.BOTTOM_MOTOR_SPEAKER_SCORE_FROM_BASE_RPMS)),
```

Comment:
Not super important but the base of the speaker is called the "Subwoofer" so you could shorten the names of these variables to something like "TOP_MOTOR_SUBWOOFER_SHOT_RPMS"